install.sh

```diff
@@ -1701,6 +1140,8 @@ AUTH_X509_FILES="src/authm_mad/remotes/x509/authenticate"
 
 AUTH_LDAP_FILES="src/authm_mad/remotes/ldap/authenticate"
 
+AUTH_SAML_FILES="src/authm_mad/remotes/saml/authenticate"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

AUTH_SAML_FILES appears unused. Verify use (or export if used externally).